### ISSUE TITLE
Kubernetes dashboard enabled with error

### ISSUE THREAD

**Issue by kiran-113** [terraform-google-modules/terraform-google-kubernetes-engine#1592]
### TL;DR

While applying the module https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/examples/safer_cluster , with https://github.com/GoogleCloudPlatform/policy-library/blob/main/samples/gke_dashboard_disable.yaml ,I am getting error KUBERNETES DASHBOARD ENABLED, but i didn't see about DASHBOARD ENABLED in the module

### Observed behavior

While applying the module https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/examples/safer_cluster , with https://github.com/GoogleCloudPlatform/policy-library/blob/main/samples/gke_dashboard_disable.yaml ,I am getting error KUBERNETES DASHBOARD ENABLED, but i didn't see about DASHBOARD ENABLED in the module

### Terraform Configuration

```hcl
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/examples/safer_cluster
```

### Terraform Version

```sh
terraform >=0.13, harshicorp/google >= 3.50.0, and policy https://github.com/GoogleCloudPlatform/policy-library/blob/main/samples/gke_dashboard_disable.yaml
```
